Apps/Playground/Scripts/experience.js

```diff
@@ -81,9 +61,8 @@ CreateBoxAsync().then(function () {
 //BABYLON.SceneLoader.AppendAsync("https://raw.githubusercontent.com/KhronosGroup/glTF-Sample-Models/master/2.0/ClearCoatTest/glTF/ClearCoatTest.gltf").then(function () {
     BABYLON.Tools.Log("Loaded");
 
-    scene.createDefaultCamera(true);
+    scene.createDefaultCamera(true, true, true);
     scene.activeCamera.alpha += Math.PI;
-    CreateInputHandling(scene);
```

Comment:
So, if we're getting rid of this stuff, can we get rid of the old `InputManager` stopgap thing altogether?

Reply:
That's the plan though I'm not sure that we can get rid of the actual code from the repo though

Reply:
@ryantrem @bghgary Do we know if InputManager (Native ver) is used in anything outside just an internal use?  If it's not, I can just delete the file.

Reply:
I doubt it is used elsewhere. It was a stopgap that @syntheticmagus wrote because we didn't have input hooked up.

Reply:
I removed the header file and cmake list reference.